### PR TITLE
Apache SuperSet and QuestDB blog post/tutorial

### PR DESCRIPTION
**Pull request by javier**
Adding blog post/tutorial for Apache Superset basic dashboards

### CODE REVIEW

**Review by javier — COMMENTED** [review]
It looks good to me

File: blog/2023-05-19-time-series-dashboards-apache-superset-and-questdb.mdx [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+  src="/img/blog/2023-05-19/create_dataset_trips_2018.png"
+  width={1056}
+/>
+
+## Creating your first chart
+
+At this point you should see a dialog to select a visualization type. If you
+clicked somewhere else you can just select the `Charts` link at the top of the
+Superset web interface and click on the name of the dataset we created on the
+previous step.
```
[the remark this answers is not in the document]
Comment (javier) [reply blog/2023-05-19-time-series-dashboards-apache-superset-and-questdb.mdx]: I mean you should be seeing exactly that. But if the user clicks somewhere else and don't know how to get back there (happened to me the first time) I am explaining how you can get back to see this screen. If you think it is not needed we can just remove the second sentence

File: blog/2023-05-19-time-series-dashboards-apache-superset-and-questdb.mdx [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+<Screenshot
+  alt="Energy Forecast chart"
+  height={469}
+  src="/img/blog/2023-05-19/energy_forecast.png"
+  width={1056}
+/>
+
+## Wrapping up
+
+Apache Superset is a very complete tool for creating data visualizations. In
```
Comment (tris0laris) [inline blog/2023-05-19-time-series-dashboards-apache-superset-and-questdb.mdx:263]: remove "very"

File: blog/2023-05-19-time-series-dashboards-apache-superset-and-questdb.mdx [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+different options depending on the chart type.
+
+At the top of this dialog, there are two tabs: one for defining the data we want
+to display, and one for customizing the appearance of the chart.
+
+For this first chart, we want to display the total number of entries, the
+average trip distance, and the average total amount paid, grouped by vendor, cab
+type, and rate code:
+
+- Add `vendor_id`, `cab_type`, and `rate_code_id` as dimensions by dragging
```
Comment (amyshwang) [inline blog/2023-05-19-time-series-dashboards-apache-superset-and-questdb.mdx:176]: I got this:
Screenshot 2023-05-19 at 17 03 39

 Two questions:
 - Do we need to tick SHOW TOTAL" (that's in your screenshot)?
 - I got

File: blog/2023-05-19-time-series-dashboards-apache-superset-and-questdb.mdx [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+<Screenshot
+  alt="Example of richer dashboard layout"
+  height={469}
+  src="/img/blog/2023-05-19/dashboard_layout.png"
+  width={1056}
+/>
+
+## Creating a dataset from a SQL statement
+
+We will now add a new dataset, but in this case, we want to use QuestDB-specific
```
Comment (amyshwang) [inline blog/2023-05-19-time-series-dashboards-apache-superset-and-questdb.mdx:217]: Screenshot 2023-05-19 at 17 10 39

**Review by javier — COMMENTED** [review]
ALL GOOD. THANKS!

**Review by amyshwang — APPROVED** [review]

**Pull request merged**